extensions/smallrye-health/runtime/src/main/java/io/quarkus/smallrye/health/runtime/SmallRyeHealthHandler.java

```diff
@@ -17,7 +20,10 @@
 
     @Override
     public void handle(RoutingContext event) {
-
+        InjectableContext context = Arc.container().getActiveContext(RequestScoped.class);
```

Comment:
@mkouba is this ok?

Reply:
Nope, it needs to be cleaned up afterwards. Also at the moment there is no way it can be active when this method is called.

Reply:
Also it's easier to use `io.quarkus.arc.ArcContainer.requestContext()` instead...

Reply:
Oh, and the method returns null if there is no request context active..

Reply:
Fixed.